src/main/flight/pid_luxfloat.c

```diff
@@ -62,6 +62,10 @@ extern float lastITermf[3], ITermLimitf[3];
 extern pt1Filter_t deltaFilter[3];
 extern pt1Filter_t yawFilter;
 
+extern bool dtermNotchInitialised, dtermBiquadLpfInitialised;
```

Comment:
same/worse problem with `deltaFilter/yawFilter etc`. When someone changes filter type in pid.c, hell will break loose.